composer.json

```diff
@@ -8,6 +8,7 @@
 		"issues": "https://github.com/Automattic/jetpack/issues"
 	},
 	"require-dev": {
+		"automattic/jetpack-changelogger": "1.1.x-dev",
```

Comment:
I don't know if this addition is necessary, since we do not use the changelogger in the root of the monorepo.

Reply:
@jeherve , @leogermani , Thanks very much for reviewing and for your feedback!  Very helpful. I've made feedback corrections. 🙂

> It looks like you'll need to run `tools/project-version.sh -u 1.13.0-alpha packages/options` to fix the failing checks.

Hmm, I ran the above command but I'm still having issues with the failing checks. 🤦‍♂️ Something to do with conflicting composer.json files?  Apologies, this is my first PR to the Jetpack plugin and I'm not understanding how these checks work or how to fix them. Any help or guidance you could provide would be hugely appreciated 😊